packages/pass-style/src/passStyleOf.js

```diff
@@ -188,13 +190,29 @@ const makePassStyleOf = passStyleHelpers => {
   return harden(passStyleOf);
 };
 
-export const passStyleOf = makePassStyleOf([
-  CopyArrayHelper,
-  CopyRecordHelper,
-  TaggedHelper,
-  ErrorHelper,
-  RemotableHelper,
-]);
+/**
+ * If there is already a `VataData` global containing a `passStyleOf`,
+ * then presumably it was endowed for us by liveslots, so we should use
+ * and export that one instead. Other software may have left it for us here,
+ * but it would require write access to our global, or the ability to
+ * provide endowments to our global, both of which seems adequate as a test of
+ * whether it is authorized to serve the same role as liveslots.
+ *
+ * NOTE HAZARD: This use by liveslots does rely on `passStyleOf` being
+ * deterministic. If it is not, then in a liveslot-like virtualized
+ * environment, it can be used to detect GC.
+ *
+ * @type {PassStyleOf}
+ */
+export const passStyleOf =
+  globalThis?.VatData?.passStyleOf ||
```

Comment:
AFAICT, this is the only mention of VatData in all of endo, and one of only two non-spurious references to liveslots (the others looking like migration cruft). If fundamental modules are going to defer to their environment, it would be good to establish an endo convention supporting that while keeping it independent rather than inverting the usual dependency direction with agoric-sdk. My first thought would be Symbol-keyed properties, e.g.
```suggestion
  globalThis?.[Symbol.for('@endo passStyleOf')] ||
```